tests/test_pipdeptree.py

```diff
@@ -488,3 +490,16 @@ def test_custom_interpreter(tmp_path, monkeypatch, capfd, args_joined):
     assert context.value.code == 1
     assert not out
     assert err == "graphviz functionality is not supported when querying" " non-host python\n"
+
+
+def test_guess_version_setuptools_not_imported(monkeypatch):
+    builtin_import = builtins.__import__
+
+    def _import(name, *args, **kwargs):
```

Comment:
This should match exactly the interface of that method and not use args, kwargs

Reply:
Got rid of it with the pytest-mock change